### PR TITLE
Date operations

### PR DESCRIPTION
**Pull request by IagoLast**
This adds new date comparison operators, still WIP

[https://gyazo.com/4b3b2bd7925a406abcbcd0071d296505](https://gyazo.com/4b3b2bd7925a406abcbcd0071d296505)

### CODE REVIEW

**Comment by davidmanzanares** [comment]
Please, provide an URL instead of a GIF, I cannot reproduce the GIF behavior

**Comment by IagoLast** [comment]
http://localhost:5000/examples/editor/#eyJhIjoibW5tYXBwbHV0b19udWxsZGF0ZXMiLCJiIjoiIiwiYyI6ImNhcnRvdmwiLCJkIjoiaHR0cHM6Ly97dXNlcn0uY2FydG8uY29tIiwiZSI6InN0cm9rZVdpZHRoOiAwXG5jb2xvcjogcmFtcChsaW5lYXIoJGJ1aWx0X3RpbWUsIDAsMSkgUFJJU00pXG5maWx0ZXI6ICRidWlsdF90aW1lID4gdGltZSgnMTk1MC0wNy0xNVQwODoxMTozMVonKSIsImYiOnsibG5nIjotNzMuOTQ0ODA1ODA0NDM1NTMsImxhdCI6NDAuNzQ1NzkwMjg1NDQ1NTR9LCJnIjoxMS4wNzY4NjY3ODU1Mzk3OTgsImgiOiJEYXJrTWF0dGVyIn0=

File: src/core/viz/expressions/time.js [hunk 1]
```diff
@@ -32,32 +32,36 @@ export default class Time extends BaseExpression {
         this.date = util.castDate(date);
         this.inlineMaker = inline => inline._impostor;
     }
-    
+
     _compile(meta) {
         this.metadata = meta;
-        const inputMin = this.metadata.columns.find(c => c.name == this.dateProperty.name).min.getTime();
-        const inputMax = this.metadata.columns.find(c => c.name == this.dateProperty.name).max.getTime();
+        const inputMin = util.castDate(this.metadata.columns.find(c => c.name).min).getTime();
```
Comment (davidmanzanares) [inline src/core/viz/expressions/time.js]: `c => c.name` This doesn't make sense, you are searching for a column and you are accepting the first one with a name.

Comment (davidmanzanares) [reply src/core/viz/expressions/time.js]: > wops, totally true, how can we get the column name from the time expression?

The parent (in this case the binary expression) should set it, check my original PR (see between)

**Pull request closed without merging**